zun/kolla-build.conf

```
@@ -1,7 +1,7 @@
 [zun-base]
 type = git
-location = git@github.com:ChameleonCloud/zun.git
-reference = chameleoncloud/train
+location = https://github.com/ChameleonCloud/zun.git
+reference = stable/train
```

```suggestion
reference = chameleoncloud/train
```

Needs to be `chameleoncloud/train` to have our patches.